implementations/rust/ockam/ockam_api/src/nodes/models/secure_channel.rs

```diff
@@ -124,3 +126,52 @@ impl<'a> DeleteSecureChannelResponse<'a> {
         }
     }
 }
+
+#[derive(Debug, Clone, Decode, Encode)]
+#[rustfmt::skip]
+#[cbor(map)]
+pub struct ShowSecureChannelRequest<'a> {
+    #[cfg(feature = "tag")]
+    #[n(0)] tag: TypeTag<1234567>,
```

Comment:
Although I can't prove that those aren't randomly generated, could you please try again?)))
This could be handy https://www.random.org/integers/?num=100&min=1000000&max=9999999&col=5&base=10&format=html&rnd=new

Reply:
Sorry, I was not aware of this procedure

Reply:
@Retamogordo no need to apologize at all. We don't have good documentation for this.